### PR TITLE
Update Import.php

### PR DESCRIPTION
**Pull request by btekbtek**
Update API request with Access Key and new URL.
To get this work please change Access Key in code to your access key from fixier.io account.

### CODE REVIEW

**Comment by sreichel** [comment]
I think API key should be set in admin config section.

**Comment by btekbtek** [comment]
Yes I know but I'm testing only now.

I found that free plan only work for base currency -  EUR so I'm not sure we continue with this. We need GBP like a base.

There is also free currency api from different service, but I have problem with implementation.
https://free.currencyconverterapi.com/api/v5/convert?q=GBP_PHP,PHP_AUD&compact=ultra

**Comment by btekbtek** [comment]
There is also free https://free.currencyconverterapi.com/api/v5/convert?q=GBP_PHP,PHP_AUD&compact=ultra 

But I have problem make this work

File: app/code/community/Philwinkle/Fixerio/Model/Import.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 class Philwinkle_Fixerio_Model_Import extends Mage_Directory_Model_Currency_Import_Abstract
 {
-    protected $_url = 'https://api.fixer.io/latest?base=%1$s&symbols=%2$s';
+    protected $_url = 'https://data.fixer.io/api/latest?access_key=YOUR_ACCESS_KEY&base=%1$s&symbols=%2$s';
```
Comment (philwinkle) [inline app/code/community/Philwinkle/Fixerio/Model/Import.php:5]: ACCESS_KEY needs to be an admin-configurable setting or have a valid fallback value. This should be store configuration-scoped. Please provide a setup script for that and we'll get this merged! Thanks!

**Review by philwinkle — CHANGES_REQUESTED** [review]
Thanks! Few updates.

**Comment by philwinkle** [comment]
@btekbtek please resubmit once `ACCESS_KEY` is admin-configurable. Thanks!

**Pull request closed without merging**